LPFilter/Source/PluginEditor.cpp

```diff
@@ -21,6 +21,8 @@ LpfilterAudioProcessorEditor::LpfilterAudioProcessorEditor (LpfilterAudioProcess
     bypassButton.addListener(this);
     bypassButton.setButtonText(processor.bypassParam->name);
     
+    startTimerHz (30);
```

Comment:
This method will trigger the timerCallback every ~33ms.
You should either call line 25(or simply call `updateComponents ` method) **before** line 24.